hangupsbot/plugins/restrictedadd.py

```diff
@@ -19,8 +19,12 @@ def __init__(self):
 def _initialise(bot):
     plugins.register_handler(_check_if_admin_added_me, type="membership")
     plugins.register_handler(_verify_botkeeper_presence, type="message")
-    plugins.register_admin_command(["allowbotadd", "removebotadd"])
+    plugins.register_admin_command(["allowbotadd", "removebotadd", "addhocheckwhitelist"])
 
+    if not bot.get_config_option('botkeeper_check_notify'):
```

Comment:
Used config as it's a global setting - if I use memory I then need to add a command to edit it, and we're trying to reduce those?